content/vault/v1.21.x (rc)/content/docs/sysadmin/snapshots/recover-a-secret/replicated-cluster.mdx

```diff
@@ -36,6 +36,37 @@ Load the snapshot holding the secret you want to recover:
 
 <Tabs>
 
+<Tab heading="Web UI" group="ui">
+
+1. Open a web browser to access the Vault UI and sign in to the root namespace.
+   <Tip title="Namespace restriction">
+
+   Snapshot load and unload operations are restricted to the root namespace. All other snapshot operations 
+   can be performed in other namespaces. 
+
+    </Tip>
+
+    <Tip title="Permissions Required">
+        This requires snapshot management permissions.
+    </Tip>
+
+2. Select **Secrets Recovery** from the left navigation menu.
+
+3. Select **Upload snapshot**.
+    ![Load snapshot button location emphasized](/img/ui-secret-recovery-upload.png)
```

Comment:
We really try to avoid multiple screenshots because of the maintenance cost. Is there a single, "hero" screenshot we can provide that we can use to make sure folks know they're on the right screen?